editor/inspector/editor_inspector.cpp

```diff
@@ -1906,6 +1906,8 @@ void EditorInspectorCategory::gui_input(const Ref<InputEvent> &p_event) {
 
 EditorInspectorCategory::EditorInspectorCategory() {
 	set_focus_mode(FOCUS_ACCESSIBILITY);
+
+	connect(SceneStringName(theme_changed), callable_mp(this, &EditorInspectorCategory::_theme_changed));
```

Comment:
This actually results in an error. Connections of self signals should be done in `NOTIFICATION_POSTINITIALIZE`.